### ISSUE TITLE
ensemble_step1.py - Missing training code

### ISSUE THREAD

**Issue by emil-varghese** [qxiaobu/FLANNEL#5]
Hi,
It appears that in the ensemble_step1.py code, some code is missing. For eg: it does not have the training steps. Is that a correct observation.
Thanks,

**Comment by qxiaobu** [comment]
yeah, training codes about step1 weren't uploaded. Because, in step 1, I just use 5 standard deep nets from pytorch lib for 4-class classification problem. Hence, I didn't supply them.
